plugins/headlamp-plugin/bin/headlamp-plugin.js

```diff
@@ -211,20 +211,72 @@ function start() {
 }
 
 /**
- * Build the plugin for production.
+ * Build the plugin package or folder of packages for production.
+ *
+ * @param packageFolder {string} - folder where the package, or folder of packages is.
  */
-function build() {
-  console.log('Watching for changes to plugin...');
+function build(packageFolder) {
+  if (!fs.existsSync(packageFolder)) {
+    console.error(`"${packageFolder}" does not exist. Not building.`);
+    return 1;
+  }
+
+  const oldCwd = process.cwd();
+  const oldBabelEnv = process.env['BABEL_ENV'];
```

Comment:
This is not being reused.

Reply:
Thanks. Now I've used it further down to restore the env to before.